integration/terraform/ec2/win/main.tf

```diff
@@ -36,41 +36,65 @@ resource "aws_instance" "cwagent" {
   vpc_security_group_ids      = [aws_security_group.ec2_security_group.id]
   associate_public_ip_address = true
   get_password_data           = true
+  user_data                   = <<EOF
+<powershell>
+Write-Output "Install OpenSSH and Firewalls which allows port 22 for connection"
```

Comment:
Curious, is this the only way to allow port 22 connection? Can this be setup upon instance launching?

Reply:
@sethAmazon didn't you update something on the AMI to allow for this?

Reply:
Yes. The only time we allow port 22 connection before launching instance would be: interact with the AMI. There are two ways for us to do that: 
* Use ImagePipeline to build the AMI and install some of the software beforehand. (1)
* Install those software when spinning up instances (3)
(1) would be not feasible for us since there are "timeout" issue (I have built it and it only concerns Deep Learning AMI for windows but not for linux). However, (3) is not the long-term solution since we rely on ImageBuilder but its share the same effect with (1); but (3) is the temporary solution I have been able to think of.